cli/js/lib.deno.ns.d.ts

```diff
@@ -18,6 +18,8 @@ declare namespace Deno {
     fn: TestFunction;
     name: string;
     skip?: boolean;
+    disableOpSanitizer?: boolean;
+    disableResourceSanitizer?: boolean;
```

Comment:
Could these just be `sanitizeOps`, `sanitizeResources` defaulting to `true`?

Reply:
Since this was probably done for implementation convenience, here's how it would look: https://github.com/nayeemrmn/deno/commit/0094e9f30ccf5506db64a5613708a6cbf9e2a256.

Reply:
Is 'sanitize' the right word here?  A new user might assume an `opSanitizer` will automatically do some clean up for them.  Alternatives could be `disableOpValidation` and `disableResourceValidation`.

Reply:
> Is 'sanitize' the right word here?
`leakCheck`? I know this is try to follow the ASAN/LSAN pattern, but plain description might be better since `sanitize` means so many things

Reply:
Nice, I prefer `leakCheck` to my suggestion.

Reply:
Other option is to use the names of sanitizers: `assertOps` and `assertResources`

Reply:
@bartlomieju I think `assertOps` might be slightly confusing: assert that ops [have some behavior], where is behavior is not described. It works for internal function names, but as API to be exposed I feel it need to be more concrete.

Reply:
@kevinkassimo I see, I'm open to suggestions; I must say it's hard for me to look at this from perspective of end-user; I've been tackling testing stuff for the past two week.

Reply:
For the record, I'm okay with sanitize. The problem with it is clear but there's no better one word alternative...

Actually, how about `leaksOps`, `leaksResources` defaulting to false? It makes perfect sense for the user to write "this test leaks ops/resources" and for the runner to complain if they haven't and it does.

Reply:
I welcome separate PRs to change the naming and add better docs/error messages after we land this one.